### DISCUSSION TITLE
How to adjust cell height margin with Jupyter on VSCode

### DISCUSSION THREAD

**Discussion by ykon550** [microsoft/vscode-jupyter#10748] [Questions and Answers] [answered]
Hi,

Can we adjust cell margin when we use Jupyter on VSCode?
I think current configuration is too spacious for me. I would like to know how to keep vertical layout tight between each cells.

Jupyter on VSCode has 16px margin between cells like following,
![Screen Shot 2022-07-09 at 22 41 58](https://user-images.githubusercontent.com/28885823/178108432-e2c99821-485c-4409-a6b4-5f25046fd8b5.png)

On the other hand, Jupyter web has only 5px padding between cells,
![Screen Shot 2022-07-09 at 22 42 07](https://user-images.githubusercontent.com/28885823/178108439-060e5953-f921-4d4c-a63b-c1fe4aa2e077.png)

Thank you for in advance.

**Comment by IanMatthewHuff** [comment, marked as answer]
Currently we don't have a setting to change this. If you want to hide the cell status bars you could set notebook.showCellStatusBar to hidden and that frees up a bit more vertical space at the cost of the cell status bars.

**Comment by RainyBlue-w** [comment]
If there is any solution now？It's really an important for me to adjust the space between cells ..
